screed/tests/test_attriberror.py

```diff
@@ -0,0 +1,82 @@
+import screed
+from screed.DBConstants import fileExtension
+import os
+
+
+class nostring:
+    def __init__(self):
+        self.exists = True
+    def __str__(self):
+        raise AttributeError
+    def __repr__(self):
+        raise AttributeError
+    def __cmp__(self):
+        raise AttributeError
```

Comment:
Yes, but we shouldn't be calling the __cmp__ function on the nostring object anywho.

Which, really means I should have used a different kind of exception, but eh.